src/lib.rs

```diff
@@ -1,4 +1,7 @@
-//! The Sprocket command line tool.
+//! [The Sprocket command line tool](https://sprocket.bio/).
```

Comment:
I figured a link would be nice here since this is what shows up at <https://docs.rs/sprocket>